src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -17,24 +17,25 @@ import { Markup } from 'interweave';
 import { Component, Fragment } from 'react';
 import myschedule from 'services/myschedule';
 import schedulecontrol from 'services/schedulecontrol';
-import { gordonColors } from 'theme';
 import urlRegex from 'url-regex-safe';
 import EditDescriptionDialog from './components/EditDescriptionDialog';
 import RemoveScheduleDialog from './components/RemoveScheduleDialog';
 import GordonScheduleCalendar from './components/ScheduleCalendar';
 import MyScheduleDialog from './components/myScheduleDialog';
+import styles from './SchedulePanel.module.css';
```

Comment:
The whole schedule panel component, including all the sub-components, is being rewritten by Gideon and Andrew, so we probably want to hold off on updating the styling of this until they are done.  Could you revert your changes within srs/components/Profile/components/SchedulePanel untill they merge their rewrite.